### ISSUE TITLE
not enough values to unpack (expected 2, got 1)

### ISSUE THREAD

**Issue by SFN1373** [prateekjoshi565/Fine-Tuning-BERT#14]
The code that reaches this part has an error.
What should I do to fix it?

![E](https://user-images.githubusercontent.com/57634421/133599630-073dbdd7-2ffc-45d7-a2bb-2f88b0e0556d.JPG)

**Comment by shibing624** [comment]
_, cls_hs = self.bert(sent_id, attention_mask=mask, return_dict=False)
